target/linux/ramips/image/mt7621.mk

```diff
@@ -529,6 +529,14 @@ define Device/gnubee_gb-pc2
 endef
 TARGET_DEVICES += gnubee_gb-pc2
 
+define Device/hilink_hlk-7621a
+  $(Device/dsa-migration)
+  DEVICE_VENDOR := HiLink
```

Comment:
If you compile this as it is with testing kernel (5.10) bootloader is not able to properly decompress the kernel ("lzma error 1"), so this line should also be added here:

$(Device/uimage-lzma-loader)

See: https://github.com/openwrt/openwrt/pull/4004